core/arch/arm/plat-stm32mp2/conf.mk

```diff
@@ -0,0 +1,62 @@
+flavor_dts_file-257F_EV1 = stm32mp257f-ev1.dts
+
+flavorlist-MP25 = $(flavor_dts_file-257F_EV1)
+
+ifneq ($(PLATFORM_FLAVOR),)
+ifeq ($(flavor_dts_file-$(PLATFORM_FLAVOR)),)
+$(error Invalid platform flavor $(PLATFORM_FLAVOR))
+endif
+CFG_EMBED_DTB_SOURCE_FILE ?= $(flavor_dts_file-$(PLATFORM_FLAVOR))
+endif
+CFG_EMBED_DTB_SOURCE_FILE ?= stm32mp257f-ev1.dts
+
+ifneq ($(filter $(CFG_EMBED_DTB_SOURCE_FILE),$(flavorlist-MP25)),)
+$(call force,CFG_STM32MP25,y)
+endif
+
+ifeq ($(filter $(CFG_STM32MP25),y),)
+$(error STM32 Platform must be defined)
+endif
+
+include core/arch/arm/cpu/cortex-armv8-0.mk
+$(call force,CFG_ARM64_core,y)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+$(call force,CFG_WITH_LPAE,y)
+supported-ta-targets ?= ta_arm64
```

Comment:
I suggest to move the above line  out of this call-force series and reorder to 3 config setting above into the below series:
```patch
 include core/arch/arm/cpu/cortex-armv8-0.mk
 supported-ta-targets ?= ta_arm64

+$(call force,CFG_ARM64_core,y)
 $(call force,CFG_DRIVERS_CLK,y)
 ...
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+$(call force,CFG_WITH_LPAE,y)